src/functions.jl

```diff
@@ -107,6 +113,18 @@ if VERSION >= v"1.1"
     end
 end
 
+@static if VERSION > v"1.9-DEV"
+    # TODO: only supports dims=: for now
```

Comment:
```suggestion
    # TODO: this will ERROR if given dims, instead of falling back to Base
    # TODO: ideally it would dispatch on the element type, for e.g. a generator of KeyedArrays
```